### DISCUSSION TITLE
Unable to commit order - empty orderKey

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#10347] [Q&A / Support] [answered]
I'm currently developing a site and am running Pimcore 10.1.3 with the Ecommerce Framework but get an issue whenever I try to commit an order. I get the error below:
![Screenshot from 2021-09-17 13-28-12](https://user-images.githubusercontent.com/5517212/133776528-e9907631-dac7-415b-9a74-750fbb341928.png)
![Screenshot from 2021-09-17 13-42-28](https://user-images.githubusercontent.com/5517212/133777062-09cc6ad6-cfe7-4cd0-a935-ec4f7428044d.png)

I've followed the entire stacktrace but couldn't find the cause of this issue. I've attached the error-log below but couldn't find anything of interest in it.
[dev.log](https://github.com/pimcore/pimcore/files/7185232/dev.log)

Do I have to add the orderKey manually? It doesn't look like that's the case in the demo.

Thanks in advance!

**Comment by fashxp** [comment, marked as answer]
did you add an order key field to your order data object? 
don't think that comes from the core.

**Reply by TS1997** [reply]
After looking through everything further that ended up being the case but it was a colleague of mine that added it and I didn't get that information
